pyaedt/application/Analysis3D.py

```diff
@@ -1118,3 +1118,160 @@ def check_intersections(output, input_list, cad_in=None):
+            indices = self._find_indices(lines, "AcDbLayerTableRecord\n")
+            for idx in indices:
+                if "2" in lines[idx + 1]:
+                    layer_names.append(lines[idx + 2].replace("\n", ""))
+            return layer_names
+
+    @pyaedt_function_handler
+    def import_dxf(
+        self,
+        file_path,
+        layers_list,
+        auto_detect_close=True,
+        self_stitch=True,
+        self_stitch_tolerance=0,
+        scale=0.001,
+        defeature_geometry=False,
+        defeature_distance=0,
+        round_coordinates=False,
+        round_num_digits=4,
+        write_poly_with_width_as_filled_poly=False,
+        import_method=1,
+        sheet_bodies_2d=True,
+    ):
+        # type: (str, list, bool, bool, float, float, bool, float, bool, int, bool, int, bool) -> bool
+        """Import dxf file.
+
+        Parameters
+        ----------
+        file_path : str
+            dxf file path.
+        layers_list : list
+            List of layer names to import.
+            If the user wants to know the layers in dxf, the method ``get_dxf_layers`` has to be called.
+        auto_detect_close : bool, optional
+            Causes polylines to be checked to see whether they are closed.
+            If a polyline is closed, the modeler creates a polygon in the design.
+            Default value is ``True``.
+        self_stitch : bool, optional
+            Causes multiple straight line segments to be joined to form polylines.
+            If the resulting polyline is closed, a polygon is created in the modeler.
+            Default value is ``True``.
+        self_stitch_tolerance : float, optional
+            Self stitch tolerance value.
+            If not provided by the user the default tolerance value is used.
+            Default value is ``0``.
+        scale : float, optional
+            Scaling factor.
+            Default value is ``0.001`` which means ``mm``.
+        defeature_geometry : bool, optional
+            Defeature geometry to reduce complexity.
+            Default value is ``False``.
+        defeature_distance : float, optional
+            Defeature tolerance distance.
+            Default value is ``0``.
+        round_coordinates : bool, optional
+            Rounds all imported data to the specified number of decimal points.
+            Default value is ``False``.
+        round_num_digits : int, optional
+            Number of digits.
+            Default value is ``4``.
+        write_poly_with_width_as_filled_poly : bool, optional
+            Imports wide polylines as polygons.
+            Default value is ``False``.
+        import_method : int, bool
+            ``Script`` or ``Acis``.
+            Default value is ``1`` which refers to ``Acis``.
+        sheet_bodies_2d : bool, optional
+            Import as 2D sheet bodies causes imported objects to be organized in terms of 2D sheets.
+            Default value is ``True``.
```

Comment:
```suggestion
        file_path : str
            Path to the DXF file.
        layers_list : list
            List of layer names to import. To get the layers in the DXF file,
            you can call the ``get_dxf_layers`` method.
        auto_detect_close : bool, optional
            Whether to check polylines to see if they are closed.
            The default is ``True``. If a polyline is closed, the modeler
            creates a polygon in the design.
        self_stitch : bool, optional
            Whether to join multiple straight line segments to form polylines.
            The default is ``True``.
        self_stitch_tolerance : float, optional
            Self stitch tolerance value. The default is ``0``.
            If not provided by the user the default tolerance value is used.
        scale : float, optional
            Scaling factor. The default is ``0.001``. The units are ``mm``.
        defeature_geometry : bool, optional
            Whether to defeature the geometry to reduce complexity.
            The default is ``False``.
        defeature_distance : float, optional
            Defeature tolerance distance. The default is ``0``.
        round_coordinates : bool, optional
            Whether to rounds all imported data to the number
            of decimal points specified by the next parameter.
            The default is ``False``.
        round_num_digits : int, optional
            Number of digits to which to round all imported data.
            The default is ``4``.
        write_poly_with_width_as_filled_poly : bool, optional
            Imports wide polylines as polygons.
            Default value is ``False``.
        import_method : int, bool
            ``Script`` or ``Acis``.
            Default value is ``1`` which refers to ``Acis``.
        sheet_bodies_2d : bool, optional
            Import as 2D sheet bodies causes imported objects to be organized in terms of 2D sheets.
            Default value is ``True``.
```

Reply:
@PipKat you are right but by "default" I meant the choice. I mean.. in AEDT if the user doesn't enable the self stich tolerance he/she has to enable the default tolerance button.